recipe/meta.yaml

```
@@ -26,9 +26,9 @@ requirements:
     - jupyterlab >=4.0.0
   run:
     - python >={{ python_min }}
-    - jupyterlab >=4.0.0
+    - jupyterlab >=4
```

Upstream now consistently uses `4.0.0` for `jupyterlab` with PR: https://github.com/rapidsai/jupyterlab-nvdashboard/pull/220

So let's keep the existing formatting

```suggestion
    - jupyterlab >=4.0.0
```